### ISSUE TITLE
Day Trades And Current Positions

### ISSUE THREAD

**Issue by patthehat033** [jmfernandes/robin_stocks#116]
Hello, I am using the functions:

r.account.get_day_trades() for getting day trades
and
r.account.get_current_positions() for getting current positions.

When I run the day trade function I get:
410 Client Error: Gone for url: https://api.robinhood.com/accounts/[AccountNumber]/recent_day_trades/
even though I have completed an option day trade. 

When I run current_positions - I am getting [] back even though I have an option position open. 

Any thoughts?

**Comment by Penlo** [comment]
The 410 Gone Error is an HTTP response status code indicating that the resource requested by the client has been permanently deleted, and that the client should not expect an alternative redirection or forwarding address.

This is not my answer, all credit goes to Google.

**Comment by patthehat033** [comment]
@Penlo Yes, I agree.  Do you know if there is another HTTP endpoint that gives day trades?

**Comment by inverseyourself** [comment]
Not sure about the option positions, but this is what I did to get your day trades.
https://github.com/jmfernandes/robin_stocks/issues/73#issuecomment-643638023
